.travis.yml

```diff
@@ -26,6 +26,7 @@ env:
   - HELLBENDER_TEST_INPUTS=gs://hellbender/test/resources/
   - HELLBENDER_TEST_STAGING=gs://hellbender/test/staging/
   - HELLBENDER_TEST_PROJECT=broad-dsde-dev
+  - HELLBENDER_JSON_SERVICE_ACCOUNT_KEY=servicekey.json
```

Comment:
Ah, now I remember why I did it this way. GOOGLE_APPLICATION_CREDENTIALS is part of the default credentials mechanism (that is, it is checked automagically for us). So if we want to test explicit credentials we have to use a different env. variable. 

I'm still switching to GOOGLE_APPLICATION_CREDENTIALS for now since it's simpler but we'll have to keep this in mind for when we set up a "logged out of Google" Travis run.